src/js/port_handler.js

```diff
@@ -1,5 +1,50 @@
 'use strict';
 
+const http = require("http");
+const bonjour = require('bonjour')();
+
+const MDNS_INTERVAL = 10000;
+const TCP_CHECK_INTERVAL = 5000;
+const TCP_TIMEOUT = 2000;
+
+const mdnsBrowser = {
+    services: [],
+};
+
+function isMobile() {
+    return ["Android", "Mac"].includes(GUI_checkOperatingSystem());
```

Comment:
why Mac is in `isMobile`?

Reply:
this is indeed wrong! I was didnt looked close and thought this is iOS, i deleted it.